WebDriverAgentLib/Categories/XCUIElement+FBTyping.m

```diff
@@ -21,6 +22,10 @@ - (BOOL)fb_typeText:(NSString *)text error:(NSError **)error
   if (!self.hasKeyboardFocus && ![self fb_tapWithError:error]) {
     return NO;
   }
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"11.0")) {
```

Comment:
I hope this will solve unexpected freezes described in https://github.com/facebook/WebDriverAgent/issues/481 and https://github.com/facebook/WebDriverAgent/issues/453 
The general idea is that the public _typeText_ method is fixed in XCTest after Xcode9 and there is no need to use undocumented features implemented by FBKeyboard anymore.